kolibri/plugins/user_profile/tasks.py

```diff
@@ -24,6 +25,15 @@
 from kolibri.utils.translation import ugettext as _
 
 
+def set_superuser(user_id):
+    new_superuser = FacilityUser.objects.get(id=user_id)
+    # make the user a new super user for this device:
+    new_superuser.facility.add_role(new_superuser, role_kinds.ADMIN)
```

Comment:
This user is brought in using a learner only sync - this means two things, they shouldn't be made a facility admin, and even when we are doing so here, it will not sync anywhere.

We should be setting them as a superuser as below, but not a facility admin.

Reply:
ah, none of the facility menus appeared after the user logins so I didn't noticed that, but it makes sense. In that case, @LianaHarris360 you can discard my previous comment, it would not make sense create a function for just one line of code.

Reply:
I updated the code so that the migrated user is a superuser and not specifically a facility admin. I am able to log in/out and see the side navigation normally. But once the server is restarted and I log back in, I have the same problem as @jredrejo, the menu items disappear. However, this issue seems to be unrelated to the code changes I've made because this also happens on the `develop` branch. I can go ahead and open a separate issue for this.

<img width="721" alt="Screenshot 2023-04-13 at 9 26 37 AM" src="https://user-images.githubusercontent.com/46411498/231800412-61180d6b-4641-4c8d-ab38-91d683906e1a.png">

Reply:
Yeah, this is not caused by your changes, definitely, @LianaHarris360 - it's either a caching issue where the side nav items are not being properly deregistered, or it's an issue with 'contamination' where plugins are being unsafely referenced from outside the plugins themselves.